client/clienterror/clienterror.go

```diff
@@ -222,6 +223,65 @@ func New(err error) *APIError {
 		}
 	}
 
+	// get v4 cluster status
+	if getClusterStatusDefaultErr, ok := err.(*clusters.GetClusterStatusDefault); ok {
```

Comment:
We talked about refactoring this before IIRC - is there a better way to do this?

Reply:
I don't know if there is.